internal/tiltfile/tiltfile_test.go

```diff
@@ -928,6 +930,49 @@ docker_build('gcr.io/some-project-162817/sancho', '.')
 		m.ImageTargetAt(0).Ref.String())
 }
 
+func TestExtraPodLabels(t *testing.T) {
+	f := newFixture(t)
+	defer f.TearDown()
+
+	f.setupExtraPodLabels("[{'baz': 'quu'}]")
```

Comment:
can you add tests for:

```
{"labelA": "valueA", "labelB": "valueB"}
[{"labelA": "valueA", "labelB": "valueB"}]
[{"labelA": "valueA"}. {"labelB": "valueB"}]
```

and assert the differences between them? i don't think the first one currently works, but think it probably should work?